### PR TITLE
Admin button functionality

### PR DESCRIPTION
**Pull request by AzukaUgbekile**
Admin button menu functionality

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -59,6 +59,30 @@ const App = () => {
...
+            />
+            <Route
+              path="/Patients"
+              element={
+                <ProtectedRoute>
+                  <Patients />
+                </ProtectedRoute>
+              }
+            />
+            <Route
```
Comment (BorisLaboratory) [inline src/App.jsx:78]: I dont think there is any editing necessary in the App.jsx. It already has all setup that is needed

File: src/pages/Admin.jsx [hunk 2]
```diff
@@ -14,81 +15,87 @@ const Admin = () => {
...
-          },
-        }}
-        size="large"
-        variant="contained"
-        startIcon={<AirlineSeatFlatAngledTwoTone />}
-      >
-        In-Service Management
-      </Button>
+      <Link to="/employees">
+        <Button
```
Comment (BorisLaboratory) [inline src/pages/Admin.jsx:19]: I think it is more efficient to use the Link inside the Button rather that outside the Button . something like       

<Button
        component={Link}
        to="/employees"

**Review by BorisLaboratory — CHANGES_REQUESTED** [review]
A few observations , please see the comments

**Comment by AzukaUgbekile** [comment]
Changes effected

File: src/pages/Admin.jsx [hunk 3]
```diff
@@ -65,6 +70,8 @@ const Admin = () => {
         Patients Management
       </Button>
       <Button
+        component={Link}
+        to="/In-Service"
```
Comment (BorisLaboratory) [inline src/pages/Admin.jsx:74]: technically , if you peep into the App.jsx "/In-Service" does not exist.
what is in the App.jsx is    "/in-service"
observe that all other routes are all lowercases. Just consistence

**Review by BorisLaboratory — CHANGES_REQUESTED** [review]
just one observation

File: src/pages/Admin.jsx (hunk 3 above)
Comment (AzukaUgbekile) [reply src/pages/Admin.jsx]: Change made

**Comment by AzukaUgbekile** [comment]
Change made

**Review by BorisLaboratory — APPROVED** [review]
Looks good

**Review by BorisLaboratory — APPROVED** [review]

**Pull request merged**